### PR TITLE
Introducing pylint configuration for robottelo

### PR DESCRIPTION
**Pull request by sthirugn**
This PR introduces pylint config and disables some pylint errors which were already widely used across robottelo.  This will help focusing on any valid errors.

### CODE REVIEW

File: tests/foreman/api/test_contentview.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # -*- encoding: utf-8 -*-
+# pylint: disable=too-many-public-methods, too-many-lines, invalid-name
```
Comment (elyezer) [inline tests/foreman/api/test_contentview.py]: I see this kind of disabling in many modules, why not creating a pylint config file? At least for global disables.

**Comment by elyezer** [comment]
My comment is not a blocker.

ACK

**Comment by sthirugn** [comment]
Investigating the config file option now

**Comment by sthirugn** [comment]
This PR is ready for review.

**Comment by elyezer** [comment]
ACK

**Comment by JacobCallahan** [comment]
ACK

**Pull request merged**